src/main/java/org/isf/medicalinventory/manager/MedicalInventoryManager.java

```diff
@@ -372,4 +386,75 @@ public void deleteInventory(MedicalInventory medicalInventory) throws OHServiceE
 		}
 		ioOperations.deleteInventory(medicalInventory);
 	}
+
+	/**
+	 * Confirm the Inventory rows of inventory.
+	 *
+	 * @param inventory - The {@link MedicalInventory}
+	 * @param inventoryRowSearchList- The list of {@link MedicalInventory}
+	 * @throws OHDataValidationException
+	 */
+	@Transactional(rollbackFor = OHServiceException.class)
+	public List<Movement> confirmMedicalInventoryRow(MedicalInventory inventory, List<MedicalInventoryRow> inventoryRowSearchList) throws OHServiceException {
+		// validate the inventory
+		this.validateMedicalInventoryRow(inventory, inventoryRowSearchList);
+		// check if inventory exist
+		int id = inventory.getId();
+		inventory = ioOperations.getInventoryById(id);
```

Comment:
why if `inventory` object is passed to the method we retrieve again from the DB? 
We don't even check if it is exising or not, so at least the comment  `// check if inventory` exist is wrong

Reply:
I didn't check it because the method validateMedicalInventoryRow()  call the methode updateMedicalInventoryRow which check if inventory exist or not before update its.

Reply:
I remove that line it is not need because we update the logic of validateInventory to not update the inventory